test/fixtures/basic_answers.yaml

```diff
@@ -1 +1,10 @@
-puppet: true
+# Format:
+# <classname>: false - don't include this class
+# <classname>: true - include and use the defaults
+# <classname>:
+#   <param>: <value> - include and override the default(s)
+#
+# See params.pp in each class for what options are available
+
+--- 
+  puppet: true
```

Comment:
ah I see, but this should happen only on tests run right? you could leave it as it was